docs/labs/installer.rst

```diff
@@ -57,133 +89,179 @@ Make a local directory for your cluster files
 =============================================
 
 The tools will create some configuration files and certificate files for your cluster.
-It is convenient to keep these in a directory, so let's make a directory on your workstation (assuming Linux or OS X) like this:
+It is convenient to keep these in a directory, so let's make a directory on your workstation like this:
 
 .. prompt:: bash $
 
     mkdir -p ~/clusters/test
     cd ~/clusters/test
 
-Later commands in this document will put some files in this directory.
+Now we'll put some files in this directory.
 
 Get some nodes
 ==============
 
-So now let's use the tools we've just installed to deploy and configure a Flocker cluster quickly!
+So now let's use the tools we've just installed to deploy and configure a Flocker cluster.
 
-Provision some machines on AWS or an OpenStack deployment (e.g. Rackspace), or even bare metal if you want to try out the experimental ZFS backend.
-Use Ubuntu 14.04, CentOS 7, or CoreOS.
+Run the following command in your ``~/clusters/test`` directory you made earlier:
 
-.. warning::
-    CoreOS support is experimental, and should not be used for production workloads.
-	ZFS support is similarly experimental.
+.. prompt:: bash $
 
-We recommend Ubuntu 14.04 if you want to try the Flocker Docker plugin.
+    mkdir terraform
+    vim terraform/terraform.tfvars
 
-Make sure you create the servers a reasonable amount of disk space, since Docker images will be stored on the VM root disk itself.
+.. note::
 
-* Use Amazon EC2 if you want to use our EBS backend.
-  **Note that VMs must be deployed in the same AZ.**
-* Use an OpenStack deployment (e.g. Rackspace, private cloud) if you want to try our OpenStack backend.
-  VMs must be deployed in the same region.
+    In the following step, do not use a key (.pem file) which is protected by a passphrase.
+    If necessary, generate and download a new keypair in the EC2 console.
 
-.. warning::
-    Make sure you can log into the nodes as **root** with a private key (e.g. on Ubuntu or CoreOS on AWS, log in as the default user, then run ``sudo mkdir /root/.ssh; sudo cp .ssh/authorized_keys /root/.ssh/authorized_keys``).
+Now paste the following variables into your ``terraform.tfvars`` file::
```

Comment:
Did not get a chance to test terraform, but when i ran unofficial flocker installer yesterday, ``cluster.yml`` had ``control_agent`` key while the script expected ``control_service``. Not sure if this problem still exists with terraform?